docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,57 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database management system you can use to store key values for Orion.
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+1. In your database, create the `store` table by using the provided DDL script, as shown in the following example: 
+
+    `psql -h HOST-NAME -U USER-NAME -d DATABASE-NAME -f ./docs/Configuring-Orion/database/postgres_storage.sql` 
+    
+2. In your database, specify the users for the `store` table and assign permissions to them.
```

Comment:
_In your database_ is repeated at the start of both steps.  Is it worth pulling it out to remove the repetition? eg: 

In your database: 

1.  Create the ....
2. Specify the ...

Reply:
Changed.